qiskit/providers/models/backendstatus.py

```diff
@@ -66,3 +74,16 @@ def __eq__(self, other):
             if self.__dict__ == other.__dict__:
                 return True
         return False
+
+    def _repr_html_(self) -> str:
+        """Return html representation of the object"""
+        rpr = self.__repr__()
+        html_code = (
+            f"<pre>{html.escape(rpr)}</pre>"
+            f"<b>name</b>: {self.backend_name}<br/>"
+            f"<b>version</b>: {self.backend_version},"
+            f" <b>pending jobs</b>: {self.pending_jobs}<br/>"
```

Comment:
Was it intentional to display these on the same line?

Reply:
Yes, to safe some space in the representation. But I can make two lines out of it if that seems better.